src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -30,7 +30,9 @@ namespace Microsoft.CodeAnalysis.CommandLine
     ///  Field Name         Type                Size (bytes)
     /// ----------------------------------------------------
     ///  Length             Integer             4
+    ///  ProtocolVersion    Integer             4
```

Comment:
It's still needed for older compilers that don't have the hash. There is existing logic to gracefully error out if the protocol version doesn't match (not that it would have ever been hit, but should do now).

Reply:
good point

---
In reply to: [215808796](https://github.com/dotnet/roslyn/pull/29711#discussion_r215808796) [](ancestors = 215808796)